src/dependencies.js

```diff
@@ -16,7 +16,7 @@
 
   // Allows calling a function when the document body is available
    function onDocumentBody(callback) {
-    if (document.body) {
+    if (Pusher.runtime.getDocument().body) {
```

Comment:
This could be moved into the runtime:

```js
if (Pusher.Runtime.isReady()) {
   ...
```

The browser implementation could do the `.body` check.